### PR TITLE
chore(docs): update to 1.9.17

### PR DESCRIPTION
**Pull request by JacobiCamunda**
Part of https://github.com/camunda/cawemo/issues/11774
Update dependencies and download for 1.9.17.

### CODE REVIEW

File: content/technical-guide/installation/supported-environments.md [hunk 1]
```diff
@@ -21,9 +21,9 @@ The following environments are supported:
 
 # Web Browser
 
-- Google Chrome latest (currently v123.0) [recommended]
-- Mozilla Firefox latest (currently v124.0)
-- Microsoft Edge latest (currently v123.0)
+- Google Chrome latest (currently v124.0) [recommended]
+- Mozilla Firefox latest (currently v125.0.1)
```
Comment (jfriedenstab) [inline content/technical-guide/installation/supported-environments.md:25]: 🙃 I think you don't need to provide the patch level here:
```suggestion
- Mozilla Firefox latest (currently v125.0)
```

File: static/download/cawemo-1.9.17.zip
Comment (jfriedenstab) [inline static/download/cawemo-1.9.17.zip:1]: ❌ The version used in the contained `.env` files and `README.txt` is still `1.9.17-alpha1` (while it should not be the alpha version).

File: content/technical-guide/third-party-libraries/cawemo/cawemo-backend.md
Comment (jfriedenstab) [inline content/technical-guide/third-party-libraries/cawemo/cawemo-backend.md:1]: ❌ The updated list doesn't seem to be correct with all those deleted dependencies.

When I run `generate-restapi-enterprise-dependency-licenses` from the `release-1.9.7` branch I get a different result.

**Review by jfriedenstab — CHANGES_REQUESTED** [review]

File: content/technical-guide/installation/supported-environments.md (hunk 1 above)
Comment (JacobiCamunda) [reply content/technical-guide/installation/supported-environments.md:25]: Okay, I looked at the [release notes](https://www.mozilla.org/en-US/firefox/125.0.1/releasenotes/), which stated: "NOTE: Due to a high-severity quality issue discovered shortly before release, the 125.0 release was skipped in favor of 125.0.1.". But it makes sense to not providing patch level here.

**Pull request merged**